### PR TITLE
add missing conf so akka can find tables

### PR DESCRIPTION
**Pull request by coreyauger**
We need to let akka know which tables to use.

### CODE REVIEW

**Comment by coreyauger** [comment]
Thanks @pvlugter that makes more sense.

**Comment by coreyauger** [comment]
Think this is what we are looking for now.

**Comment by coreyauger** [comment]
doh! .. ok hows this?

**Comment by pvlugter** [comment]
I've made some changes in #396 so that the tests pass. I've rebased this PR.

File: proxy/spanner/src/main/resources/application.conf [hunk 1]
```diff
@@ -10,15 +10,29 @@ akka {
   }
 
   persistence {
-    journal {
-      plugin = "akka.persistence.spanner.journal"
-    }
-
```
Comment (pvlugter) [inline proxy/spanner/src/main/resources/application.conf]: I think we still need this plugin setting. I'll add it back.

File: proxy/spanner/src/main/resources/application.conf [hunk 2]
```diff
@@ -19,6 +19,24 @@ akka {
...
+
+        # name of the auxilary table used to store information about deletes
+        deletions-table = "deletions"
+        deletions-table = "deletions_"${?SPANNER_TABLE_PREFIX}
+      }
+
+      snapshot {
+        table = "snapshots"
+        table = "snapshots_"${?SPANNER_TABLE_PREFIX}
+      }
```
Comment (pvlugter) [inline proxy/spanner/src/main/resources/application.conf:39]: All these table names are also defined further down the file under `cloudstate.proxy.spanner`. Maybe we should have the table names only defined once, and use setting references?

Comment (coreyauger) [reply proxy/spanner/src/main/resources/application.conf:39]: That makes sense to me.

Comment (pvlugter) [reply proxy/spanner/src/main/resources/application.conf:39]: Also, these tables will be called `journal_`, `tags_`, and so on, when `SPANNER_TABLE_PREFIX` is not set. The initial setting will always be overridden given the string prefix (different from if it was just `table = ${?TABLE}`. I don't know if that matters (maybe table names can't end with an underscore?) if the prefix var is always set — although could then be better to make it required.

Comment (coreyauger) [reply proxy/spanner/src/main/resources/application.conf:39]: Lets loop @hseeberger in here... I believe these are his names.

Comment (pvlugter) [reply proxy/spanner/src/main/resources/application.conf:34]: Okay. Have updated to reference the `cloudstate.proxy.spanner` settings, like for project, instance, database. Removed the redundant settings.

**Comment by hseeberger** [comment]
@coreyauger and @pvlugter, nice job!

File: proxy/spanner/src/main/resources/application.conf (hunk 2 above)
Comment (hseeberger) [reply proxy/spanner/src/main/resources/application.conf:34]: I'm not sure about the potentially trailing underscore. Let's get this merged quickly to fix the issue at hand and then deal with that underscore later.

**Pull request merged**